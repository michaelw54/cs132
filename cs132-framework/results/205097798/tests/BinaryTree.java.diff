1,4c1,26
< ParseException: Encountered " <IDENTIFIER> "w_2 "" at line 2, column 1.
< Was expecting:
<     "return" ...
<     
---
> 16
> 100000000
> 8
> 16
> 4
> 8
> 12
> 14
> 16
> 20
> 24
> 28
> 1
> 1
> 1
> 0
> 1
> 4
> 8
> 14
> 16
> 20
> 24
> 28
> 0
> 0
