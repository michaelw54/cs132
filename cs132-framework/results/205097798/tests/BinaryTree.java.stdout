func Main()
w_2 = alloc(4)
w_3 = alloc(4)
[w_2 + 0] = w_3
[w_3 + 0] = @BT_Start
w_1 = @BT_Start
w_0 = call w_1(w_2)
print(w_0)
ret_ = 0
return ret_

func BT_Start(this)
root = 0
ntb = 0
nti = 0
java.lang.NullPointerException
